OpenEdXMobile/res/values/strings.xml

```diff
@@ -453,7 +453,7 @@
     <!-- Error shown when user attempts to send email, but has no email app -->
     <string name="email_client_not_present">No e-mail clients installed</string>
     <!-- Search for a course prompt -->
-    <string name="search_for_courses">I want to learn about...</string>
+    <string name="search_for_courses" tools:ignore="MissingTranslation">Search for courses</string>
```

Comment:
@catong are we good to go with this string?

Reply:
"Search for courses" looks good to me. 👍